From c752a4e0e1983ebf3018a39b6f9dcbda26d997c9 Mon Sep 17 00:00:00 2001
From: Benda Xu <heroxbd@gentoo.org>
Date: Fri, 24 Oct 2014 18:26:54 +0900
Subject: donot install test and demonstration executables

casacore_{assay,floatcheck} are removed and only used with scons.
measuresdata.csh and ms2uvfits should be included in casacore-doc as examples.
change measuresdata.csh shebang to /usr/bin/tcsh as in Debian.
---
 measures/apps/CMakeLists.txt                | 2 +-
 measures/apps/measuresdata/measuresdata.csh | 2 +-
 msfits/apps/CMakeLists.txt                  | 5 -----
 scons-tools/CMakeLists.txt                  | 5 -----
 4 files changed, 2 insertions(+), 12 deletions(-)
 delete mode 100644 msfits/apps/CMakeLists.txt
 delete mode 100644 scons-tools/CMakeLists.txt

diff --git a/measures/apps/CMakeLists.txt b/measures/apps/CMakeLists.txt
index 1901418..b888c9d 100644
--- a/measures/apps/CMakeLists.txt
+++ b/measures/apps/CMakeLists.txt
@@ -7,4 +7,4 @@ endforeach(prog findmeastable)
 add_executable (measuresdata  measuresdata/measuresdata.cc)
 target_link_libraries (measuresdata casa_measures)
 install(TARGETS measuresdata DESTINATION bin)
-install(PROGRAMS measuresdata/measuresdata.csh DESTINATION bin)
+install(PROGRAMS DESTINATION bin)
diff --git a/measures/apps/measuresdata/measuresdata.csh b/measures/apps/measuresdata/measuresdata.csh
index c6fe6b0..b0e8260 100755
--- a/measures/apps/measuresdata/measuresdata.csh
+++ b/measures/apps/measuresdata/measuresdata.csh
@@ -1,4 +1,4 @@
-#!/bin/tcsh
+#!/usr/bin/tcsh
 # 
 # measuresdata.csh
 # Must be executable. Call it with all defaults or arguments for measuresdata
diff --git a/msfits/apps/CMakeLists.txt b/msfits/apps/CMakeLists.txt
deleted file mode 100644
index d57af6d..0000000
--- a/msfits/apps/CMakeLists.txt
+++ /dev/null
@@ -1,5 +0,0 @@
-foreach(prog ms2uvfits)
-    add_executable (${prog}  ${prog}.cc)
-    target_link_libraries (${prog} casa_msfits)
-    install(TARGETS ${prog} DESTINATION bin)
-endforeach(prog ms2uvfits)
diff --git a/scons-tools/CMakeLists.txt b/scons-tools/CMakeLists.txt
deleted file mode 100644
index 3fdfe09..0000000
--- a/scons-tools/CMakeLists.txt
+++ /dev/null
@@ -1,5 +0,0 @@
-install(PROGRAMS
-  casacore_assay
-  casacore_floatcheck
-DESTINATION bin
-)
